lib/acl.c

```diff
@@ -179,7 +204,7 @@ process_ipv6_acl(struct gatekeeper_if *iface, unsigned int lcore_id,
 	if (unlikely(ret < 0)) {
 		RTE_LOG(ERR, ACL,
 			"invalid arguments given to rte_acl_classify()\n");
-		drop_ipv6_acl_pkts(acl->mbufs, acl->num, iface);
+		drop_ipv6_acl_pkts(acl->mbufs, acl->num);
 		acl->num = 0;
```

Comment:
Since this function has the only two calls to `drop_ipv6_acl_pkts()`, maybe we can just remove that function now? And in this function, have a `goto` statement that on these errors frees the packets, sets `acl->num` to 0, and returns either 0 or return value on error.